.bazelrc

```diff
@@ -437,8 +437,8 @@ build:avx_win --copt=/arch:AVX
 # Use Clang-cl compiler on Windows
 build:win_clang --copt=/clang:-Weverything
 build:win_clang --extra_toolchains=@local_config_cc//:cc-toolchain-x64_windows-clang-cl
-build:win_clang --extra_execution_platforms=//tensorflow/tools/toolchains/win:x64_windows-clang-cl
-build:win_clang --host_platform=//tensorflow/tools/toolchains/win:x64_windows-clang-cl
+build:win_clang --extra_execution_platforms=//tools/toolchains/win:x64_windows-clang-cl
+build:win_clang --host_platform=//tools/toolchains/win:x64_windows-clang-cl
```

Comment:
@ddunl Could you please help take a look if these .bazelrc changes are okay?

Reply:
LGTM!

Reply:
Thank you! :)

Reply:
On second thought I realized that because we still use TF's bazelrc, we may need to have these under a new config like `win_clang_xla`. Does that sound ok?

Reply:
Hi @ddunl, got your point. I will create a pull request to TF and update TF's .bazelrc with win_clang_xla config and revert the changes (back to build:win_clang --extra_execution_platforms=//tensorflow/tools/toolchains/win:x64_windows-clang-cl
) in XLA bazelrc. How often TF bazelrc is merged to XLA bazelrc? Do you have any documentation for XLA where we can update the steps to run XLA on Windows?

Reply:
I can merge the TF change with this change on our end, I just wanted to make sure that we can use a different config for XLA. I think right now we unfortunately just have to have an extra copy like `win_clang_xla` that has all the same options but just the correct toolchain paths for this repo. If that sounds reasonable then I can do that change with this PR internally.

Reply:
Yeah, that sounds good. Please go ahead and create win_clang_xla with this PR internally